src/components/ButtonPanel.js

```diff
@@ -0,0 +1,47 @@
+
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => (
+  <div className="panel">
+    <div className="group-1">
+      <Button name="AC" />
+      <Button name="plus-minus" />
+      <Button name="percent" />
+      <Button name="division" />
+    </div>
+
+    <div className="group-2">
+      <Button name="7" />
+      <Button name="8" />
+      <Button name="9" />
+      <Button name="X" />
+
```

Comment:
You might wanna remove this line, makes your code scattered and also practice good indentation in the rest of the code 👍

Reply:
Thanks for the detailed comments! This was very helpful. Will  work on it :)